### ISSUE TITLE
Make books private by default, require complete metadata before they can be 'published'

### ISSUE THREAD

**Issue by SteelWagstaff** [pressbooks/ideas#463]
Currently, creators can make books public or private when initially creating them. However, this often results in publicly available content which lacks important metadata. We can help authors/publishers ensure they fulfill their obligations to provide accurate, useful metadata by making books private by default and making all or some of the following fields required before a book can be made public:

-     title
-     author (or editor)
-     publisher
-     copyright date (I think its copyright year in PB)
-     publication date
-     License [already required]
-     Short description
-     Edition

We should also consider adding some form of contact method for readers to contact the publishers directly with questions, concerns, and requests. See https://github.com/pressbooks/ideas/issues/268

Related: https://github.com/pressbooks/ideas/issues/427

**Comment by arzola** [comment]
I like this idea specially taking into account now we are publishing an OPDS feed that requires metadata to be present. cc @cmurtagh

**Comment by cmurtagh** [comment]
Yeah, I like this too. @SteelWagstaff do you feel we could do this now, or would it be better to wait until we had some assistance in completing the metadata? What would the reaction be from Network Managers and authors if we didn't wait?

**Comment by SteelWagstaff** [comment]
I think this would be a definite no go for clients, as currently written, in part because many clients allow instructors to make simple teaching and learning material public. These books are used by students but don't necessarily have a 'publisher' or an 'edition'. If we reduced the metadata fields to 'only the essentials' -- I think it might be feasible. It would be easiest if it applied to newly created books at first -- i.e. the default PB behaviour is all new books start private and can only be made public if required metadata has been supplied. As long as it didn't impact books that are already public, I can't see people getting too upset about this.

**Comment by cmurtagh** [comment]
I wasn't suggesting that we make books with incomplete metadata private, or change the status of any existing book, only prevent a book from being made public until it had the required metadata. What are 'only the essentials'?

**Comment by SteelWagstaff** [comment]
I'd say they are:
1. Title
2. Author (or editor)
3. Copyright date (year -- default value as date of creation)
4. Copyright License (default value: All Rights Reserved)
5. Publication date (default value could be date book was made public?)

Metadata that is _highly recommended_ but probably should not be *required*:
1. Cover image
2. Description (short and/or long) 
3. Publisher
4. Relevant subject term(s)

**Comment by jcurtis-pressbooks** [comment]
I have a few concerns about rushing into this item without considering our bigger-picture goals for improving metadata and our strategy for achieving these goals. 

For example: 
**Metadata standards:** We know we want to create MARC records and that's an important priority for UK expansion. Is the list of essential metadata in this task the same as what would be needed for MARC records? Are there any other common standards (e.g. OPDS) that we should consult to make sure what we deem "essential" will serve other objectives for data-sharing and quality of metadata output?

**User experience:** Michelle is just beginning user research on how to improve the whole metadata user experience and Book Info metadata. Yet if we make this type of change, we should do it informed by what changes to the UX may be important in order to make it very easy to complete the essential metadata fields. We should also be thoughtful about how we design the user notifications if someone thinks they are publishing a book but they have omitted one of the essential metadata fields. How will that work so that they know exactly what the blocker is and how to address it? 

**Client communication & input:** This is significant change to how making books public happens, and it's something we need to communicate with clients about. Ideally we would have given them a heads up at the Product Update that we are thinking about this (and why), sharing the proposed changes, and then ask: What are your thoughts about this? Any concerns? That would ensure we're making a policy and practical change with due input from clients and due notice that it's coming. They will need to communicate with their users and potentially change the support/training materials they provide. We will also need to change our support and training materials (including videos) about the basics of making a book public. 

I'm concerned we are rushing into this task without some of the context and supportive steps that will ensure we're doing the right things in the right order RE metadata.

**Comment by cmurtagh** [comment]
I'm  100% with you on this Julie. My goal this sprint was to size and talk about this task as is, as I suspect it will lead to further discussion and need refinement.

**Comment by jcurtis-pressbooks** [comment]
Thanks, Christopher, that makes sense. Please surface any "product management" related questions or items that need further clarification or research so that we can make sure the teams are aligned on how we want to make metadata work better in Pressbooks.

**Comment by cmurtagh** [comment]
This phase DoD:

- [x] internal Tech discussion, and bring our ideas to the Steering Committee to talk about a cohesive meta data strategy and polices around it

**Comment by cmurtagh** [comment]
New DoD for next phase:

- [ ] New (empty) books will be private by default instead of public

**Comment by arzola** [comment]
this would remove the option to make a book public upon book creation, now books created everywhere will be private by default

https://github.com/pressbooks/pressbooks/pull/4256

**Comment by tw77** [comment]
The option to make a book public upon creation is successfully removed by that PR, on dev networks, so that specific change is ready for release.
